Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,3 +1,28 @@
-.small-card {
-  width: 50%;
+.mat-card{
+  display:flex;
+  flex-direction: column;
 }
+
+// mat-card-footer doesn't align with bottom, you have to user manually styling
+// --> https://github.com/angular/components/issues/11094
+.mat-card-header {
+  flex-shrink: 0;
+}
+  
+.mat-card-content{
+  flex-grow: 1;
+  overflow: auto;
```

Comment:
What are the for? They do nothing for me, as I tested in chrome.

Reply:
if there is no flex-grow, the button in actions is moving up

Reply:
if there is no flex-grow, the button is sliding up

Reply:
Actually you can orientate on the other mat-cards styling and there used classes.
I am not sure but maybe `pb-center-element` will fix the button-problem.

Reply:
Ok, as you have wrote beforehand this really is a bug. Please add the fix to the `overrides.scss`
https://github.com/angular/components/issues/661

Reply:
Please add it to the overrides.scss so it will apply to all `mat-cards`.